makefiles/S3.toml

```diff
@@ -33,15 +33,22 @@ args = [
 
 [tasks.publish-fluvio]
 dependencies = [
-    "build-release",
+    "build-prod",
     "install-fluvio-package",
     "publish-fluvio-env",
 ]
-script = "${HOME}/.fluvio/bin/fluvio package --test publish --version=${FLUVIO_VERSION} ${FORCE_RELEASE} ./target/${TARGET}/release/fluvio"
+env = { FLUVIO_VERSION = { script = ["cat VERSION"] } }
+script = """
+${HOME}/.fluvio/extensions/fluvio-package \
+    ${FLUVIO_PUBLISH_TEST} \
+    publish \
+    --version=${FLUVIO_VERSION} \
+    ${FORCE_RELEASE} \
+    ./target/${TARGET}/release/fluvio
```

Comment:
Why not `cargo run` instead of referring target directory?

Reply:
What do you mean? This command is naming the executable file that should be published, it is not invoking it.

Reply:
got it